src/parser/core/modules/ActionTimeline/ActionTimeline.tsx

```diff
@@ -82,13 +87,36 @@ export class ActionTimeline extends Analyser {
 		this.addEventHook('complete', this.onComplete)
 	}
 
+	protected resolveRowLabel(config: InternalRowConfig) {
+		for (const maybeAction of config.content) {
+			let fullAction = undefined
+			if (typeof maybeAction === 'object') {
+				fullAction = maybeAction
+			} else if (typeof maybeAction === 'number') {
+				fullAction = this.data.getAction(maybeAction)
+			} else if (typeof maybeAction === 'string') {
+				fullAction = this.data.actions[maybeAction as ActionKey]
+			}
+			if (fullAction === undefined || fullAction === null) { continue }
```

Comment:
Minor nit on this one: this is the one case where the == sign is ok to use. ```fullAction == null``` will do a check for null or undefined so you don't need to explicitly look for both

Reply:
TIL, that makes sense! Patched.